en/introduction.md

```diff
@@ -21,6 +21,7 @@ Recommended practices are organized into two primary sections
 * The server hosting GTFS Realtime data should be reliable and consistently return validly-formatted protobuf-encoded responses. Fewer than 1% of responses should be invalid (protobuf errors or fetching errors).
 * The web-server hosting GTFS Realtime data should be configured to correctly report the file modification date (see HTTP/1.1 - Request for Comments 2616, under Section 14.29) so consumers can leverage the `If-Modified-Since` HTTP header. This saves producers and consumers bandwidth by avoiding transferring feed contents that haven't changed.
 * Feeds should provide protocol buffer-encoded feed content by default when queried via an HTTP request at the given URL - consumers should not need to define special HTTP accept headers to receive protocol-buffer encoded content.
+* Feeds should provide an option to display data in a structured UTF-8 text output (e.g., JSON, Protobuf [TextFormat](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/TextFormat)) in addition to the protocol-buffer encoded feed content (e.g., by appending `&debug` to the feed URL, providing an alternate URL to different file such as `trip-updates.json`). This enables a simple human-readable version of the information for troubleshooting purposes without requiring special tools to view the current contents of the real-time feed.
```

Comment:
```suggestion
* Feeds should provide an option to request their content in a structured UTF-8 text representation (e.g., JSON, [Protobuf TextFormat](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/TextFormat)) in addition to the protocol-buffer encoded feed content (e.g., by appending `&debug` to the feed URL, providing an alternate URL pathname such as `trip-updates.pb.json`). This enables opting into a simple human-readable version of the information for troubleshooting purposes, without requiring special tools to view the binary-encoded contents of the real-time feed. An appropriate `Content-Type` should be sent to denote which text representation is being served.
```

Reply:
Bonus points for serving with >=1 `Link: <…>; rel="alternate"; type="…"` HTTP headers to let clients know where the alternative represenation(s) are!